test/test.js

```diff
@@ -1765,3 +1765,30 @@ test( "Validation triggered on radio and checkbox via click", function() {
 	// test if there is no error anymore
 	ok( form.find( "input.error" ).length === 0, "Form valid" );
 });
+
+test( "destruction of Validator", function() {
+	expect( 2 );
+
+	var form = $( "#validatorDestroy" ),
+		validate = form.validate();
+
+	form.on( "submit.validate", function( event )
+	{
+		event.preventDefault();
+
+		ok( !!$( this ).data( "validator" ), "Validator has events and object" );
+	});
+
+	form.submit();
+
+	validate.destroy();
+
+	form.on( "submit.validate", function( event )
```

Comment:
This doesn't make sense. You're just binding another event handler, for no good reason in the same namespace. Could run the same `ok()` assertion after `destroy()` call, no need for a submit. Also applies above, you don't need to submit the form at all to check the `data()`.